Sources/TokamakCore/Tokens/ColorScheme.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum ColorScheme: CaseIterable {
+  case dark
+  case light
+}
+
+struct ColorSchemeKey: EnvironmentKey {
+  static let defaultValue: ColorScheme = .light
```

Comment:
Would it be possible for this to match the value of the `(prefers-color-scheme: dark)` media query? You could create the query with `window.matchMedia('(prefers-color-scheme: dark)')` and add an event listener to update this value on change.

Reply:
Yes, that's what I'm working right now. I think it would make sense to add the listener in an `onAppear` closure on the new `DOMEnvironment` view introduced in this PR, but this obviously needs a working `onAppear` implementation first, which is what I'm working on right now 🙂